app/views/home/contributors.md

```diff
@@ -0,0 +1,69 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that might be looking for ways to use those skills on an open source project.
```

Comment:
I think potentially if we can start developing guidelines for other skillsets, then it would be useful to start including them as the target for this. But for now, I think scoping it to people with existing software skills is the right way to go. This matches up with our objectives, as currently defined.

Other skills might be:
- technical writing
- project management
- branding
- marketing
- visual design
- user experience design
- ...

For each of these things, we might consider if there are already resources that go in depth. If so, should we create a guide for that topic? Then we could link to that guide if we want to start working towards encouraging other skill sets.